csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -2,14 +2,15 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Runtime.InteropServices;
 using System.Text;
 using System.Runtime.InteropServices;
 using System.IO;
 
 namespace Microsoft.ML.OnnxRuntime
 {
     /// <summary>
-    /// TODO Add documentation about which optimizations are enabled for each value.
+    /// Graph optimization level to use with SessionOptions
```

Comment:
Missed this earlier. Can point to https://github.com/microsoft/onnxruntime/blob/master/docs/ONNX_Runtime_Graph_Optimizations.md.